packages/hURL/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=hurl
+pkgver=20.afca9c5
+pkgrel=1
+pkgdesc='Hexadecimal & URL (en/de)coder.'
+arch=('any')
+groups=('blackarch' 'blackarch-misc' 'blackarch-exploitation')
+url='https://github.com/fnord0/hURL'
+license=('custom:unknown')
+depends=('perl' 'perl-cgi' 'perl-uri' 'perl-digest-sha1' 'perl-mime-types')
+makedepends=('git')
+source=("$pkgname::git+https://github.com/fnord0/hURL.git")
+sha512sums=('SKIP')
+
+prepare() {
```

Comment:
The correct name for this function `pkgver()`

Reply:
yes my bad. Copy from pkgbuilds --